package.json

```diff
@@ -59,14 +58,22 @@
     "ldapjs": "1.0.1"
   },
   "devDependencies": {
+    "babel-core": "6.21.0",
+    "babel-loader": "6.2.10",
+    "babel-preset-es2015": "6.18.0",
     "chai": "3.5.0",
     "eslint": "3.12.2",
     "font-awesome": "4.7.0",
     "handlebars": "4.0.6",
```

Comment:
Nah, that was Handlebars used by the helpers tests, but the version in repo, i.e. [the one currently used by the client is v4.0.5](https://github.com/thelounge/lounge/blob/master/client/js/libs/handlebars.js#L3).